### ISSUE TITLE
Entire custom ad notification should be clickable

### ISSUE THREAD

**Issue by tmancey** [brave/brave-browser#13137]
## Steps to Reproduce

   1. View ad notification
   2. Click ad notification outside the text area

## Actual result:

Click event is not fired

## Expected result:
Click event should be fired

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? No
- Can you reproduce this issue with the nightly channel? No

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? N/A
- Does the issue resolve itself when disabling Brave Rewards? N/A
- Is the issue reproducible on the latest version of Chrome? N/A

## Miscellaneous Information:

Confirm working as expected on touch devices such as Windows Surface

**Comment by tmancey** [comment]
Resolved as part of #13132, please test as part of the new custom ad notifications

**Comment by btlechowski** [comment]
Verification passed on

Brave | 1.26.50 Chromium: 91.0.4472.77&nbsp;(Official Build)&nbsp;beta&nbsp;(64-bit)
-- | --
Revision | 1cecd5c8a856bc2a5adda436e7b84d8d21b339b6-refs/branch-heads/4472@{#1246}
OS | Ubuntu 18.04 LTS

Verified the custom notification is clickable in 
- top-left corner
- top-right corner
- bottom-left corner
- bottom-right corner
- center
- on title text
- on ad content text
- on Ads icon

Verified clicking on the above opened landing page. 
Verified that I got an unblinded token for `view`
```
[3724:3724:0603/195656.603666:VERBOSE1:confirmations.cc(248)] Successfully redeemed unblinded token with confirmation id f1bdf6de-0625-4444-a3bc-ea38605ef571, creative instance id 4cda3ae7-1f70-4bca-b361-ca01d6a8f1ed and view
```
Verified that I got an unblinded token for `clicked`
```
[3724:3724:0603/195659.259079:VERBOSE1:confirmations.cc(248)] Successfully redeemed unblinded token with confirmation id 402c126c-2f42-4cd6-b026-685285b9674e, creative instance id 4cda3ae7-1f70-4bca-b361-ca01d6a8f1ed and click
```

Verified that I got an unblinded token for `landed`
```
[3724:3724:0603/195712.749576:VERBOSE1:confirmations.cc(248)] Successfully redeemed unblinded token with confirmation id e6da63d2-d962-4750-85c0-6c81b503ef93, creative instance id 4cda3ae7-1f70-4bca-b361-ca01d6a8f1ed and landed
```

Verified clicking on `x` dismissed the ad. The landing page was not opened and unblinded token for dismissed was redeemed:
```
[504:504:0603/205641.927396:VERBOSE1:confirmations.cc(248)] Successfully redeemed unblinded token with confirmation id d7bd998e-f721-428b-8f40-bbeeadaeb208, creative instance id 4cda3ae7-1f70-4bca-b361-ca01d6a8f1ed and dismiss
```
Logged https://github.com/brave/brave-browser/issues/16230 for dismiss area being too big
